CMakeLists.txt

```diff
@@ -451,10 +451,7 @@ endif()
 include_directories(external/rapidjson/include external/easylogging++ src contrib/epee/include external external/supercop/include)
 
 if(APPLE)
-  include_directories(SYSTEM /usr/include/malloc)
```

Comment:
If it was declared as SYSTEM include, then it means, that the intention was to suppress some compiler warnings coming from there. But since the directory doesn't exist, it doesn't matter anymore.